sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/xml/TestCase.java

```diff
@@ -23,234 +23,85 @@
 import com.thoughtworks.xstream.annotations.XStreamAsAttribute;
 import com.thoughtworks.xstream.annotations.XStreamOmitField;
 import org.apache.commons.lang.StringUtils;
+import org.apache.commons.lang.builder.ToStringBuilder;
+import org.sonar.api.internal.google.common.annotations.VisibleForTesting;
 
-/**
- * The Class TestCase.
- */
 @XStreamAlias("testcase")
 public final class TestCase {
```

Comment:
Only indirect. I did not want to create to many accessors on these classes since XStream does not need them, but I'll create a couple to write this test.